tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.Tests/ActionTests.cs

```diff
@@ -368,6 +369,31 @@ public async Task Action_SendTaskModuleUrlResponseError()
             await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer);
         }
 
+        [Fact]
+        public async Task Action_SendTabCardResponseError()
+        {
+            await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer);
+        }
+        
+        [Fact]
+        public async Task Action_SendTabCardResponse()
+        {
+            await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer);
+        }
+        
+        [Fact]
+        public async Task Action_SendTabAuthResponseErrorWithAdapter()
+        {
+            await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer);
+        }
+
+        //[Fact]
+        //public async Task Action_SendTabAuthResponseError()
+        //{
+        //      NOTE: Current test adapter is not a BotFrameworkAdapter,
+        //    await TestUtils.RunTestScript(_resourceExplorerFixture.ResourceExplorer);
+        //}
```

Comment:
Why leave this here?

Reply:
For when the test adapter supports mocking these results (these same .json script tests exist in javascript sdk also)